x/blob/types/events.go

```diff
@@ -6,7 +6,7 @@ import (
 
 var (
 	EventTypePayForBlob       = proto.MessageName(&EventPayForBlobs{})
-	EventTypeUpdateBlobParams = proto.MessageName(&EventPayForBlobs{})
+	EventTypeUpdateBlobParams = proto.MessageName(&EventUpdateBlobParams{})
```

Comment:
luckily, this event is not used anywhere and I think we can delete it. what do you think? @rootulp